src/ex_02/file_stats.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'kaspar lilja'
+__email__ = 'kalilja@nmbu.no'
+
+
+def char_counts(textfilename):
+    file_name = open(textfilename)
+    str_text = file_name.read()
+    file_name.close()
+    count_ = {}
```

Comment:
This should be a list of length 256, not a dictionary. Read the assignment more properly next time.